vector/src/main/java/im/vector/app/core/platform/VectorBaseFragment.kt

```diff
@@ -222,6 +223,10 @@ abstract class VectorBaseFragment<VB : ViewBinding> : Fragment(), MavericksView
         }
     }
 
+    //we need to decide if we want to use vectorBaseActivity here
+    //the problem is that it casts getActivity() result to non nullable type, so in case if it will return null, NullPointerException will be thrown
+    protected fun configureToolbar(toolbar: MaterialToolbar) = (activity as? VectorBaseActivity<*>)?.configureToolbar(toolbar) ?: ToolbarConfig(null, toolbar)
```

Comment:
it's possible to remove toolbar argument if we will use same id for toolbar on every screen. But in that case we'll need to look it up with findviewById, which is slower that just pass reference from bindings. On the other side passing toolbar view every time you want to update title on a screen could be annoying boilerplate.